target/linux/ramips/image/mt7621.mk

```diff
@@ -63,6 +63,14 @@ define Device/dir-860l-b1
 endef
 TARGET_DEVICES += dir-860l-b1
 
+define Device/afoundry-ew1200
+  DTS := AFOUNDRY-EW1200
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := AFOUNDRY EW1200
+  DEVICE_PACKAGES := kmod-usb3 kmod-ledtrig-usbport kmod-ata-core kmod-ata-ahci
```

Comment:
Ops, I see my copy&paste mistake, sorry for that!
But anyway, you should us **kmod-usb-ledtrig-usbport**, not kmod-ledtrig-{usbdev,usbport}.